versions.yaml

```diff
@@ -243,6 +243,14 @@ externals:
     url: "https://github.com/containerd/nydus-snapshotter"
     version: "v0.1.0"
 
+  virtiofsd:
+    description: "vhost-user virtio-fs device backend written in Rust"
+    url: "https://gitlab.com/virtio-fs/virtiofsd"
+    version: v1.2.0
+    meta:
+      # yamllint disable-line rule:line-length
+      binary: "https://gitlab.com/virtio-fs/virtiofsd/uploads/b26d9891caac83209a3fdd24bcf2ae86/virtiofsd-v1.2.0.zip"
```

Comment:
Also, pulling the binary is not consistent with the title of the PR, which states "Build" 😄

Reply:
> So you pull the binary instead of rebuilding it?

Yep, we pull the binary for the architectures that have the binary, otherwise it'll be built, and that will have to be done for different architectures.

> So maybe, to facilitate later updates, add a comment here explaining how you pick up the binary path:

Sure, I can add that.


I will do after I finish running the CI on the tests side, which depends on this one, just to be sure I don't break things there.

Reply:
Updated!